File: aplication.js
// Sample JavaScript Program
// Author: Your Name

// Function to greet a user
function greetUser(name) {
    return `Hello, ${name}! Welcome to JavaScript.`;
}

// Call the function
let userName = "Alex";
console.log(greetUser(userName));

// Basic math operation
function multiply(a, b) {
    return a * b;
}

console.log("5 × 4 =", multiply(5, 4));

// Working with an array
const fruits = ["Apple", "Banana", "Mango"];
fruits.push("Orange");
console.log("Fruit List:", fruits);

// Simple object with a method
const person = {
    name: "Sara",
    age: 30,
    sayHello() {
        console.log(`Hi! I'm ${this.name}.`);
    }
};

person.sayHello();

// Loop example
console.log("Counting:");
for (let i = 1; i <= 5; i++) {
    console.log(i);
}

File: aplication.java
// Sample JavaScript Program
// Author: Your Name

// Function to greet a user
function greetUser(name) {
    return `Hello, ${name}! Welcome to JavaScript.`;
}

// Example: Using the function
let userName = "John";
console.log(greetUser(userName));

// Basic arithmetic function
function addNumbers(a, b) {
    return a + b;
}

let sum = addNumbers(10, 20);
console.log("The sum is:", sum);

// Working with arrays
let colors = ["Red", "Green", "Blue"];
colors.push("Yellow");
console.log("Color List:", colors);

// Simple object
let person = {
    name: "Alice",
    age: 25,
    isStudent: true,
    greet() {
        console.log(`Hi, I'm ${this.name} and I'm ${this.age} years old.`);
    }
};

person.greet();

